### DISCUSSION TITLE
[BUG]ValueError: Start client failed, or server is not set

### DISCUSSION THREAD

**Discussion by electroboyz** [Chia-Network/chia-blockchain#9236] [Support]
```
2021-11-05T05:33:44.631 full_node chia.rpc.rpc_server     : WARNING  Error while handling message: Traceback (most recent call last):
  File "chia\rpc\rpc_server.py", line 81, in inner
  File "chia\rpc\rpc_server.py", line 161, in open_connection
ValueError: Start client failed, or server is not set
```

Repeated error on Windows 10, I guess this is about DB again. 
Chia seems to be the most buggy mining software in the world of cryptocurrency. Dev should seriously look into this corrupted DB issue

**Comment by loppefaaret** [comment]
i guess it isn't about the DB - this is usually an indication about a module not getting fed the address of an endpoint - a connection attempt unable to resolve a hostname for instance - but i see this is stemming from the rpc_server - so my guess is the daemon service has failed before this error ? 
any change you could tell me what version you are experiencing this with ? 
and potentially add the `debug.log` found under your users folder `~\.chia\mainnet\log\debug.log` as well

**Comment by electroboyz** [comment]
it's the DB of course, cuz when I started without DB it started syncing fine from scratch but I had to put my backed up DB in place as I dont wanna wait half a week to sync from scratch and got the error I posted. Tried with Chia 1.2.9 and 1.2.10
